README.md

```diff
@@ -6,18 +6,44 @@ Content Sources is an application for storing information about external content
 
 ##Developing
 
-Create a `.env` file with:
-~~~
+### Configuring Postgresql
+
+As root run: 
+```shell
+yum install -y postgresql
+
+echo "host    all             all             127.0.0.1/32            trust" >> /var/lib/pgsql/data/pg_hba.conf
+echo "host    all             all             ::1/128                 trust" >> /var/lib/pgsql/data/pg_hba.conf
+systemctl start postgresql
+systemctl enable postgresql
+sudo -u postgres createdb content
+sudo -u postgres psql -c "CREATE USER content2 WITH PASSWORD 'content'"
+sudo -u postgres psql -c "grant all privileges on database content TO content"
```

Comment:
grant all privileges on database content TO content2

Reply:
fixed :)